src/lambda/events/generate_test_event.py

```diff
@@ -42,81 +45,111 @@
     "UnsubscribeURL": "https://sns.us-east-1.amazonaws.com/?Action=Unsubscribe&SubscriptionArn=arn:aws:sns:us-east-1:563295687221:phil_s3_bucket_update:89b889dd-9e68-41a5-b865-c194f09d024c"
 }
 
-def create_message_template(project_id, folder_id):
+def create_message_template(entity):
   return {
     "appId": "example-app-1",
-    "recordId": "-4I2GOqDSdjaXsbuw8oYXBKK",
+    "recordId": entity.annotations.recordId.pop(),
     "record": {
-      "parentProjectId": project_id,
-      "rawFolderId": folder_id,
-      "fileEntityId": "",
-      "s3Bucket": "",
-      "s3Key": ""
+      "parentProjectId": "syn0000000",
```

Comment:
Does it matter this is syn000000? I think you can do entity.projectId?

Reply:
`projectId` is a default view (dataset) column, but I pass the entity to this function so I don't have access to that information. If we actually used `parentProjectId` we could add a parameter to this function and/or iterate through the dataset rows rather than the entities. But that makes the preceding logic (line 135 `test_data = ...`) a little messier.